test/android-manifest-specs.js

```diff
@@ -2,22 +2,65 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import 'mochawait';
 import ADB from '../lib/adb.js';
+import path from 'path';
+import _fs from 'fs';
+import B from 'bluebird';
+import { exec } from 'teen_process';
+import { util } from 'appium-support';
+
+const fs = {
+  mkdir: B.promisify(_fs.mkdir),
+  writeFile: B.promisify(_fs.writeFile),
+  readFile: B.promisify(_fs.readFile),
+  unlink: B.promisify(_fs.unlink)
+};
 
 chai.use(chaiAsPromised);
 
 describe('Android-manifest', async () => {
-  it('packageAndLaunchActivityFromManifest', async () => {
-    let adb = new ADB();
+  let adb = new ADB();
+  before(async () => {
     await adb.createADB();
+
+  });
+  it('should correctly parse packageAndLaunchActivityFromManifest', async () => {
     let {apkPackage, apkActivity} = await adb.packageAndLaunchActivityFromManifest('./test/ContactManager.apk');
     apkPackage.should.be.equal('com.example.android.contactmanager');
     apkActivity.should.be.equal('com.example.android.contactmanager.ContactManager');
   });
+  it('should correctly have internet permission', async () => {
+    let flag = await adb.hasInternetPermissionFromManifest('./test/ContactManager-selendroid.apk');
+    flag.should.be.true;
+  });
+  it('should correctly not have internet permission', async () => {
+    let flag = await adb.hasInternetPermissionFromManifest('./test/ContactManager.apk');
+    flag.should.be.false;
+  });
+  it('should compile and insert manifest', async () => {
+    let appPackage    = 'com.example.android.contactmanager',
```

Comment:
i know it kinda looks pretty, but we don't line things up on = in lists like this. i think we should either do it everywhere or nowhere